packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -280,6 +289,46 @@ export default class CashierStore extends BaseStore {
             this.setSessionTimeout(false);
             this.setTimeoutCashierUrl();
         }
+        await BinarySocket.wait('get_account_status');
+        this.checkCashierLock();
+        if (this.active_container === this.config.deposit.container) {
+            this.checkDepositLock();
+        }
+        if (this.active_container === this.config.withdraw.container) {
+            this.checkWithdrawalLock();
+        }
+    }
+
+    @action.bound
+    checkCashierLock() {
```

Comment:
can we do all of these in cashier-store? why are they needed in client-store?

Reply:
because i got it from account status, i am afraid in the future that it can be used somewhere else, so i keep anything that belongs to client status to client store instead

Reply:
my bad, I thought this was client-store file, nvm